pages/index.tsx

```diff
@@ -57,10 +57,12 @@ export default function Home() {
     setPlacesInfo(prevVal => prevVal.results)
   }
 
-  const callApi = async (point: string) => {
+ 
+
+  const callApi = async (typeOfInterest: string) => {
     const params = {
-      city: 'Paris',
-      point
+      city: location ? `${location.city}`: 'Paris',
```

Comment:
[possible boulder] Didn't we set the default lat and lng to somewhere in london? 
Either we switch Paris to london here, or we change our default lat and lng to paris

Otherwise, our user would be seeing london but get recommendations for paris

Reply:
You are correct, I changed the default to london.